js/index.js

```diff
@@ -48,5 +73,46 @@ function createCardGrid() {
 function getRandomIndex() {
   return Math.floor(Math.random() * imagePickList.length);
 }
-
 createCardGrid();
+
+//how much time has passed since you first clicked on a card.
+let startTime;
+let timerInterval;
+let clickedCards = 0;
+
+function startTimer() {
+  if (!startTime) {
+    startTime = Date.now();
+    timerInterval = setInterval(updateTimer, 1000);
+  }
+
+  // Check if all cards have been clicked
+  if (clickedCards === cardCount) {
+    clearInterval(timerInterval);
+  }
+}
+
+function updateTimer() {
+  const currentTime = Date.now();
+  const timePassed = Math.floor((currentTime - startTime) / 1000);
+  timerDisplay.innerText = `Timer: ${timePassed} seconds`;
+}
+
+//Reset Button
+
+function resetGame() {
+  clearInterval(timerInterval);
+  startTime = null;
+  clickedCards = 0;
+  timerDisplay.innerText = "Timer: 0 seconds";
```

Comment:
This is nice, however, it would be more conventional and perhaps nicer if the timer followed a mm:ss format 🙏🏽 
But it is up to you!

Reply:
done, thanks